src/lib/platform/MSWindowsDesks.cpp

```diff
@@ -602,6 +606,16 @@ MSWindowsDesks::deskThread(void* vdesk)
 {
     MSG msg;
 
+	BOOL vistaOrGreater = FALSE;
```

Comment:
I think it would be best to extract this chunk of code to a separate function, for example `isVistaOrGreater`

Reply:
Agreed. There exist version helper functions for Windows, would it be better to use those instead? https://docs.microsoft.com/en-us/windows/win32/api/versionhelpers/nf-versionhelpers-iswindowsvistaorgreater

Reply:
Indeed, a standard helper in this case makes sense.

Reply:
Done!